ga4gh/serverconfig.py

```diff
@@ -26,6 +26,7 @@ class BaseConfig(object):
     SIMULATED_BACKEND_VARIANT_DENSITY = 0.5
     SIMULATED_BACKEND_NUM_VARIANT_SETS = 1
 
+    CACHE_MAX_SIZE = 50
```

Comment:
this should be renamed to specify which cache it is, since we might have more than one and "cache" is ambiguous

what about `FILE_HANDLE_CACHE_MAX_SIZE`?